kyo-prelude/shared/src/main/scala/kyo/debug/Debug.scala

```diff
@@ -0,0 +1,51 @@
+package kyo
+
+import Ansi.*
+import kyo.kernel.Effect
+import kyo.kernel.Safepoint
+
+object Debug:
+
+    def apply[A, S](v: => A < S)(using frame: Frame): A < S =
```

Comment:
I decided to use `println` directly and avoid introducing suspensions so it's easy to use the effect anywhere. For example, if the method had `IO` suspensions, adding `Debug` to a computation would require the user to add the `IO` effect if it's not already present. I think it's ok since it's meant only for debugging.

Reply:
I also considered using `Log` but it's common for users to have trouble with logs not appearing due to missing configs.

Reply:
Alternatively, you could also consider using logging via java.util.logging. It's available everywhere, doesn't bring any dependency and is interoperable.